helm/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: apim
 # Also update CHANGELOG.md
-version: 4.0.0
+version: 4.0.1
 appVersion: 4.0.0
```

Comment:
@kamiiiel Does the app version need to change?

Reply:
Don't think so, this appVersion represent the default value used to pull the images when the chart is deployed. So 4.0.0 looks good to me as we are on the master branch.

Reply:
So we don't change the chart version?